MastodonSDK/Sources/MastodonAsset/Generated/Assets.swift

```diff
@@ -159,6 +159,7 @@ public enum Asset {
       public static let peopleAdd = ImageAsset(name: "Scene/Compose/people.add")
       public static let pollFill = ImageAsset(name: "Scene/Compose/poll.fill")
       public static let poll = ImageAsset(name: "Scene/Compose/poll")
+      public static let questionmarkCircle = ImageAsset(name: "Scene/Compose/questionmark.circle")
```

Comment:
As far as I can tell, all of the icons here are custom ones. I think you should be able to swap to use the built-in SF Symbol (especially since the relevant symbol has been available since iOS 13)

Reply:
This is also a custom one, it’s just similar to the SF Symbol. What is the general preference, to use mostly custom icons or mostly SF Symbols? One SF Symbol here would probably look inconsistent with the rest.

Reply:
Oh nice, I didn’t notice. Very cool!